Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
I think the goal in the end, in the next major is to have **no** `array_flip` at all. This means that there should be flipped colors + a call to `array_flip` in the deprecated path of `getAll()`, exactly like you did. @core23 , please comment on that.